README.md

```diff
@@ -609,6 +609,7 @@ Often times, you find yourself having to re-write regular expressions for common
 | method                      | description                                                                                                                 |
 | --------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
 | `boolean`                   | Match a boolean value (using equality)                                                                                      |
+| `string`                    | Match a string value (using equality)                                                                                       |
```

Comment:
This won't match on equality, but rather type. I'd suggest leaving as "Match a string value"

Reply:
You're right. Done!